win32/CMakeLists.txt

```diff
@@ -19,15 +21,18 @@ set(pal_win32_c_files
     src/string_utils.c
     src/timer.c
     src/sync_win32.c
+    src/file_win32.c
 )
 
 FILE(GLOB pal_win32_md_files "devdoc/*.md")
 SOURCE_GROUP(devdoc FILES ${pal_win32_md_files})
 
 include_directories(${CMAKE_CURRENT_LIST_DIR}/inc)
+include_directories(../common/inc)
 
 add_library(pal_win32 ${pal_win32_h_files} ${pal_win32_c_files} ${pal_win32_md_files})
-target_link_libraries(pal_win32 pal_interfaces pal_common ws2_32 synchronization rpcrt4 azure_c_logging)
+target_link_libraries(pal_win32 pal_interfaces pal_internal ws2_32 synchronization rpcrt4 azure_c_logging)
 target_include_directories(pal_win32 PUBLIC ${CMAKE_CURRENT_LIST_DIR}/inc)
 
 add_subdirectory(tests)
+add_subdirectory(../common/tests common/tests)
```

Comment:
>common [](start = 20, length = 6)

Should just leave this in the common CMakeLists.txt

Reply:
Discussed offline

---
In reply to: [451914167](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451914167) [](ancestors = 451914167)